### PR TITLE
Fix/ois/filter tree

### PR DESCRIPTION
**Pull request by oyvindis**
Jira issue link: [link](LINK_HERE)

> check applicable boxes

- [ ] I have made tests to match the user stories
- [ ] This code does not require tests that match user stories

### CODE REVIEW

File: applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx [hunk 1]
```diff
@@ -30,18 +30,29 @@ const hasSomeChildren = node =>
 const hasSomeSiblingChildren = siblings =>
   Array.isArray(siblings) && siblings.some(hasSomeChildren);
 
+const isActiveFilter = (isMultiselectFilterValue, activeFilter, key) => {
+  let isActive = false;
+  if (isMultiselectFilterValue && activeFilter) {
+    const filterArray = activeFilter.split(',');
+    filterArray.forEach(filter => {
```
Comment (markkopaas) [inline applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

**Review by markkopaas — APPROVED** [review]

File: applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx (hunk 1 above)
Comment (markkopaas) [reply applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx]: enda kortere er lodash:
_.some(users, { 'user': 'barney'});

Comment (markkopaas) [reply applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

File: applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx [hunk 2]
```diff
@@ -30,18 +30,29 @@ const hasSomeChildren = node =>
...
+      if (filter === key) {
+        isActive = true;
+      }
+    });
+  } else if (activeFilter === key) {
```
Comment (markkopaas) [inline applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx]: du kan bruke early exit i stedet av else

File: applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx (hunk 1 above)
Comment (oyvindis) [reply applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx]: egentlig kunne jeg forenkle til:
if (activeFilter && activeFilter.includes(key)) {
    isActive = true;
  }

File: applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx (hunk 2 above)
Comment (oyvindis) [reply applications/search/src/pages/search-page/filter-tree/filter-tree.component.jsx]: rettet av forrige forenkling

**Pull request merged**